CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This file is used to list changes made in each version of the sql_server cookboo
 
 ## Unreleased
 
+- limit ci integration parallel jobs to avoid Vagrant image download rate-limiting errors - [@jhboricua](https://github.com/jhboricua)
```

Comment:
```suggestion
- Limit test kitchen parallel jobs to avoid Vagrant image download rate-limiting errors - [@jhboricua](https://github.com/jhboricua)
```

Reply:
I've updated the changelog per your suggestion.